api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/AutoMLConfiguration.scala

```diff
@@ -16,22 +16,25 @@ trait AutoMLConfiguration extends AlgorithmConfigurations {
       ("H2OAutoMLStoppingCriteriaParams", classOf[AutoMLStoppingCriteriaV99], classOf[AutoMLStoppingCriteria], Getter),
       ("H2OAutoMLBuildModelsParams", classOf[AutoMLBuildModelsV99], classOf[AutoMLBuildModels], Field))
 
+    val leaderboardFrame =
```

Comment:
please see bellow - could we just make it transient?

Reply:
It's not exposed on the MOJO model and and pipeline.load() resets the value to `null`.